package.json

```diff
@@ -16,27 +30,24 @@
     "react-notification-badge": "^1.3.4",
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.1.5",
-    "typeface-roboto": "0.0.54"
-  },
-  "scripts": {
-    "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
-    "test": "react-app-rewired test --env=jsdom",
-    "eject": "react-app-rewired eject",
-    "lint": "eslint src/ --ext .js",
-    "lint-functions": "cd functions && npm run lint"
+    "typeface-roboto": "0.0.54",
+    "webpack": "^3.11.0"
```

Comment:
This fixes another build issue I found in TravisCI related to the internals of CRA.